database/sql/V048__create_settlement_rewards.sql

```diff
@@ -3,7 +3,9 @@
 CREATE TABLE settlement_scores (
   auction_id bigint PRIMARY KEY,
   winning_score numeric(78,0) NOT NULL,
-  reference_score numeric(78,0) NOT NULL
+  reference_score numeric(78,0) NOT NULL,
```

Comment:
What happens if only a single solver offers a valid solution?
Should we maybe keep `reference_score` nullable for that case instead of using a default value?
Also a comment on what the `reference_score` is would be nice.

Reply:
Added comment for reference_score.